tests/lib/rules/require-expect.js

```diff
@@ -294,6 +305,35 @@ ruleTester.run("require-expect", rule, {
             code: "test('name', function(assert) { other.expect(1); assert.ok(true); });",
             options: [],
             errors: [{ message: alwaysErrorMessage("assert.expect") }]
+        },
+
+        // "always" configuration - errors are reported on `test()` line
```

Comment:
There is some overlap, but the tests are for slightly different purposes. Plus this test has a `module()` declaration. So I am happy to leave the test in here, even if it seems redundant. Sound okay?